data/templates/default.context

```diff
@@ -27,7 +27,6 @@ $endif$
 % make chapter, section bookmarks visible when opening document
 \placebookmarks[chapter, section, subsection, subsubsection, subsubsubsection, subsubsubsubsection][chapter, section]
 \setupinteractionscreen[option=bookmark]
-\setuptagging[state=start]
```

Comment:
this is not needed anymore?

Reply:
No, the tagging seems to be controlled by the `setupstructure` setting, regardless of whether or not the output is PDF/A.